nativelib/src/main/resources/scala-native/time_millis.cpp

```diff
@@ -0,0 +1,44 @@
+#if defined(_WIN32)
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#else
+#include <stdio.h>
+#include <sys/time.h>
+#endif
+
+extern "C" {
+long long scalanative_current_time_millis() {
```

Comment:
Does this file really need to be C++?

Reply:
Good point, it probably can be regular C